pkg/queue/health/health_state.go

```diff
@@ -101,11 +104,16 @@ func (h *State) HandleHealthProbe(prober func() bool, isAggressive bool, w http.
 		io.WriteString(w, notAliveBody)
 	}
 
+	sendShuttingDown := func() {
+		w.WriteHeader(http.StatusConflict)
```

Comment:
Maybe `BadRequest`?

Reply:
`BadRequest` indicated request was structured wrong. Probably 503? 
Also comment for variable `shuttingDownBody` mentions its meant for 503 response body.

Reply:
503 is used for sending not alive as well. Might not be best idea to use 503 to indicate multiple states.

Reply:
`410 Gone` seems quite good, if you squint only a relatively small amount: `This response is sent when the requested content has been permanently deleted from server, with no forwarding address`. Having said that the pedant in me does feel like this is really a server-side failure so should be a 5-- code. 

Probably the most "correct" option is to use 503 for both and use [the `Retry-After:` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503) (or a similar header) to indicate whether it's worth retrying.

I'm also a fan of 418 for this use case ;).

Reply:
(this seems like it could easily become a bike shed tho so just to be very clear: I don’t think it matters very much and all options seem fine to me!)

Reply:
I definitely thought about 418 :). I think I like 410 Gone better, if everyone is ok with that.